main.tf

```diff
@@ -43,6 +43,19 @@ resource "aws_cloudfront_distribution" "api_gate" {
     }
   }
 
+origin {
+    domain_name = var.ae_mdw_fqdn
+    origin_id   = "ae_mdw"
```

Comment:
let's be consistent with the naming, all other behaviours have `api_` prefix.

Reply:
dd8edb9